### ISSUE TITLE
(due 9/15) Regenerate warp-cloth with GMM from FW-GAN-VVT test set

### ISSUE THREAD

**Issue by andrewjong** [andrewjong/ShineOn-Virtual-Tryon#106]
Realized GMM trained on FW-GAN alone overfits and does not generalize well. Switch back to the checkpoint we previously trained on 3 datasets: viton, mpv, and vvt
